src/Compilers/CSharp/Portable/Binder/Binder_Crefs.cs

```diff
@@ -349,8 +352,19 @@ private ImmutableArray<Symbol> ComputeSortedCrefMembers(NamespaceOrTypeSymbol? c
                     diagnose: false,
                     useSiteInfo: ref useSiteInfo);
 
+                if (memberNameText is "nint" or "nuint"
+                    && containerOpt is null
+                    && arity == 0
+                    && !hasParameterList
+                    && !IsViableType(result))
+                {
+                    result.Free(); // Won't be using this.
+                    CheckFeatureAvailability(syntax, MessageID.IDS_FeatureNativeInt, diagnostics);
+                    builder = ArrayBuilder<Symbol>.GetInstance();
+                    builder.Add(this.GetSpecialType(memberName == "nint" ? SpecialType.System_IntPtr : SpecialType.System_UIntPtr, diagnostics, syntax).AsNativeInteger());
```

Comment:
This should also likely be `memberNameText`? A test for `nuint` that would catch this would be useful as well.